contracts/interface/ILiquidityDex.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface ILiquidityDex {
+    function doSwap(
+        uint256 _sellAmount,
+        uint256 _minBuyAmount,
+        address _liquidator,

```

Comment:
liquidator?

Reply:
In the original design, it was "spender," and I thought the spender should always be the UL in our use case. So I changed it to "_liquidator," but after our discussion, I think this parameter can be removed.